### ISSUE TITLE
Duplicated information in documentation

### ISSUE THREAD

**Issue by nsheff** [databio/COCOA#34]
Related to #33 

I found a lot of documentation to be duplicated, which made it hard for me to find the information I needed quickly. It makes it hard to quickly get started with COCOA. Here is an example, where the return value of the `convertToFromNullDist` function is documented with a pretty complex paragraph describing multiple layers of things:

```
#' @return a list of data.frames. If given a list where each item is 
#' a data.frame with results from one COCOA permutation, this function
#' will return a list of data.frames where each data.frame contains the
#' null distributions for a single region set. The output data.frames will
#' have the same columns as the input data.frames. If given a list where each
#' item is a data.frame with the null distribution/s for a single region
#' set, this function will return a list where each item is a data.frame
#' with one row for each region set (e.g. a data.frame with results for
#' a single COCOA permutation).
```

Besides just being hard to parse and understand in general, a lot of this is simply duplicated from the function description. Maybe the function needs to be simplified, if it really requires this level of detail.

**Comment by kinganimz** [comment]
if `@return` is the same, for several functions - do you duplicate the description, put it once and refer to it, or just outline the action? - I'll think about it

**Comment by kinganimz** [comment]
Descriptions have been simplified
